unlock-app/package.json

```diff
@@ -45,21 +42,14 @@
     "react-dom": "^16.8.3",
     "react-jazzicon": "^0.1.2",
     "react-redux": "^6.0.1",
-    "react-router-dom": "^4.2.2",
-    "react-router-redux": "^5.0.0-alpha.9",
     "react-showdown": "^1.6.0",
     "react-test-renderer": "16.8.3",
     "react-testing-library": "^6.0.0",
     "redux": "^4.0.1",
-    "rollup": "^1.6.0",
-    "rollup-plugin-babel": "^4.3.2",
-    "rollup-plugin-terser": "^4.0.4",
     "rss": "^1.2.2",
     "run-script-os": "^1.0.5",
     "storybook-chromatic": "^1.3.0",
-    "storybook-react-router": "^1.0.2",
```

Comment:
could you explain why this is not needed any longer? The last time I checked, storybook crashed if it wasn't present

Reply:
I think we moved away a little while ago from that router... to use `connected-next-router`.
I tested Storybook and it works as expected..